gooddata-sdk/gooddata_sdk/compute/model/execution.py

```diff
@@ -10,18 +11,43 @@
 from gooddata_sdk.compute.model.metric import Metric
 
 
+@dataclass
```

Comment:
Please use attrs instead of built-in data classes.

Reply:
fixed in last commit